### PR TITLE
[Enhancement] add idref to Fedora and RHEL 6 & 7 XCCDF content

### LINKED ISSUE #609: Perform XCCDF substitution also for RHEL-7 and Fedora content
[SSG commit 3df0d3043144ee07064d78775866c1c54543f87b](https://github.com/OpenSCAP/scap-security-guide/commit/3df0d3043144ee07064d78775866c1c54543f87b) performed the XCCDF substitution for RHEL-6 content (IOW instead of using hard-coded XCCDF variable values it started to use ```XCCDF variable substitution``` feature that got introduced in OpenSCAP version 1.0.8.

The very same change should be performed for existing RHEL-7 and Fedora content (or even content in other benchmarks / products that is currently using hardcoded XCCDF variable values instead of actual ```sub idref="particular_xccdf_variable_name"``` constructs).

### PR DESCRIPTION
**Pull request by redhatrises**
- Add selinux XCCDF content to Fedora
- Add account expiration XCCDF content to Fedora
- Add audit flush content to Fedora
- Update some idref instances in RHEL6
- Fixes #609

### CODE REVIEW

**Comment by mpreisler** [comment]
This seems to be doing more than advertised. Adding entire new groups and new rules.

**Comment by redhatrises** [comment]
@mpreisler true. Updated the comment section.

File: Fedora/input/system/accounts/pam.xml [hunk 1]
```diff
@@ -575,7 +597,7 @@ The DoD and FISMA requirement is 24 passwords.</description>
 To verify the password reuse setting is compliant, run the following command:
 <pre>$ grep remember /etc/pam.d/system-auth</pre>
 The output should show the following at the end of the line:
-<pre>remember=24</pre>
+<pre>remember=<sub idref="var_password_pam_unix_rememer" /></pre>
```
Comment (mpreisler) [inline Fedora/input/system/accounts/pam.xml]: typo: rememer

**Comment by redhatrises** [comment]
@mpreisler nice catch. Fixed.

**Comment by mpreisler** [comment]
ACK. Very nice changes, kudos!

**Pull request merged**